distributed/http/static/css/status.css

```diff
@@ -6,45 +6,53 @@
 @media (min-width: 0px) {
   #status-fluid {
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 3fr 1fr;
+    grid-template-rows: 1fr 2fr 6fr 2fr;
```

Comment:
FIXME the new cluster-wide RAM usage widget gets completely invisible if the browser window is too short. Need to find a way to set a minimum absolute height.

Reply:
cc @ian-r-rose

Reply:
```suggestion
    grid-template-rows: 2fr 2fr 6fr 2fr;
```